rubocop-rspec.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |spec|
   spec.test_files = spec.files.grep(%r{^spec/})
   spec.extra_rdoc_files = ['MIT-LICENSE.md', 'README.md']
 
-  spec.add_runtime_dependency 'rubocop', '>= 0.42.0'
+  spec.add_runtime_dependency 'rubocop', '>= 0.49.0'
```

Comment:
We might consider making this `~> 0.49` or have a `< 1.0` constraint also. Not that it's likely to happen soon, but would be slightly more correct.